irohad/validation/impl/stateless_validator_impl.cpp

```diff
@@ -20,15 +20,18 @@
 #include <chrono>
 #include <utility>
```

Comment:
Check if we can easily remove old stateless validator. If so, do it and mark [this](https://soramitsu.atlassian.net/browse/IR-1149) task as done